migrations/20210312233834-BetterFailedRetweetStatus.ts

```diff
@@ -0,0 +1,13 @@
+import { Db } from 'mongodb';
+import { TweetEntity } from '../src/entity/tweet';
+
+module.exports = {
+  async up(db: Db): Promise<void> {
+    await db.collection<TweetEntity>('tweet').updateMany({}, { $set: { failedAt: undefined, retweetedAt: undefined } });
```

Comment:
This is confusing, could you please create a newline for each steps including the query?

Reply:
Also, will this cause a trouble for tweets that have already been sent?

Reply:
> Also, will this cause a trouble for tweets that have already been sent?

Could you please elaborate on possible troubles that may arise? I was trying to add new column to current colletion and mark scheduled tweets with `hasRetweeted` set as true to fill the `retweetedAt` column. I'm not really familiar with Mongo, so I need your advice.

Reply:
Say, I have a tweet with `{ tweetId: 'random' }` that has been retweeted already, so `retweetedAt` will be `retweetedAt: <date>`.

The bot will periodically check the database of any unretweeted tweets, which is fulfilled when `retweetedAt != null`. Therefore, the bot will attempt to retweet the said tweet, which shouldn't be possible as the account has already retweeted that.

Reply:
Whoops, seems I misread the code somewhere. Accepted.